src/addrman.h

```diff
@@ -304,8 +337,13 @@ class CAddrMan
 
         unsigned char nVersion = 1;
```

Comment:
Bump `nVersion` if `-asmap` is set?

Reply:
I would say `nVersion` says something about backwards compatibility, but at first glance it doesn't look like the current version rejects a higher `nVersion`.